### ISSUE TITLE
TypeCastException in expectedDataset CONTAINS operation

### ISSUE THREAD

**Issue by rmpestano** [database-rider/database-rider#176]
see @ericjturley comment:

https://github.com/database-rider/database-rider/commit/9ae686f1492400b931afacc25ce69f3db29fa03c#commitcomment-36452783

**Comment by rmpestano** [comment]
Have you had the chance to try with `1.9.1-SNAPSHOT` @ericjturley? just repeating my coment here in case you miss the one in the commit itself.

To enable snapshots just add this maven repo on you pom.xml:

```
<repositories>
    <repository>
        <snapshots/>
        <id>snapshots</id>
        <name>libs-snapshot</name>
        <url>https://oss.sonatype.org/content/repositories/snapshots</url>
    </repository>
</repositories>
```

**Comment by rmpestano** [comment]
The fix was released in `v1.9.1`. if you still find the issue, please reopen this issue with a reproducer if possible.

Thanks for the feedback!
